src/eicr_anonymization/anonymizer.py

```diff
@@ -88,11 +139,25 @@ def _normalize_value(value: str):
 class Anonymizer:
     """Anonymizes the data."""
 
-    def __init__(self, seed: int | None = None):
-        """Initialize the Anonymizer class."""
-        if seed is not None:
-            # Set the seed for reproducibility
-            random.seed(seed)
+    def __init__(
+        self,
+        reproducible: bool | int | None = False,
+        deterministic_functions: bool = False,
+    ):
+        """Initialize the Anonymizer class.
+
+        Args:
+            reproducible (bool | int): If True, the random seed will be set to a fixed value for
```

Comment:
I think separating out the seed and reproducible is a bit cleaner - it feels a bit mushy putting those into one field